### PR TITLE
add process wide env variable to msbuildsdkspath

### PR DESCRIPTION
**Pull request by rohit21agrawal**
Turns out dotnet.exe we use to launch the test sets an environment variable called "MSBuildSdksPath" which is read by the other dotnet.exe that we use to launch pack. Not sure when this started happening, but for now i am setting a process wide environment variable to override this behavior.

Fixes part of https://github.com/NuGet/Client.Engineering/issues/8 
CC: @nkolev92 @emgarten

### CODE REVIEW

**Review by emgarten — APPROVED** [review]

**Comment by nkolev92** [comment]
I have reenabled xplat tests too. Will wait for the tests to pass on the CI and then we can merge.

**Review by nkolev92 — APPROVED** [review]

File: test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj [hunk 1]
```diff
@@ -35,7 +34,7 @@
   </ItemGroup>
   <Import Project="$(BuildCommonDirectory)common.targets" />
   <PropertyGroup>
-    <PostBuildEvent>xcopy /y $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Microsoft.CSharp.Core.targets $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Roslyn\</PostBuildEvent>
+    <PostBuildEvent Condition=" '$(IsXPlat)' != 'true' ">xcopy /y $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Microsoft.CSharp.Core.targets $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Roslyn\</PostBuildEvent>
```
Comment (rohit21agrawal) [inline test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj]: use the copy task instead to make it cross plat : https://msdn.microsoft.com/en-us/library/3e54c37h.aspx

Comment (nkolev92) [reply test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj]: Updated.

File: test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj [hunk 2]
```diff
@@ -33,8 +33,8 @@
     </PackageReference>
   </ItemGroup>
   <Import Project="$(BuildCommonDirectory)common.targets" />
-  <PropertyGroup>
-    <PostBuildEvent Condition=" '$(IsXPlat)' != 'true' ">xcopy /y $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Microsoft.CSharp.Core.targets $(MSBuildProjectDirectory)\bin\$(Configuration)\$(OutputPath)$(TargetFramework)\$(RuntimeIdentifier)\Roslyn\</PostBuildEvent>
-  </PropertyGroup>
+  <Target Name="AfterBuild">
```
Comment (rohit21agrawal) [inline test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj]: change this to ``` <Target Name="XCopyTargets" AfterTargets="AfterBuild" />```

**Pull request merged**